src/Tribe/Main.php

```diff
@@ -329,7 +329,8 @@ public function add_hooks() {
 		add_filter( 'body_class', array( $this, 'add_js_class' ) );
 		add_action( 'wp_footer', array( $this, 'toggle_js_class' ) );
 
-		add_filter( 'cron_schedules', array( tribe( 'cron' ), 'filter_cron_schedules' ) );
+		// Schedule cron if needed
+		add_action( 'admin_init', array( tribe( 'cron' ), 'schedule' ) );
```

Comment:
Need feedback on which action this should run on, is admin_init good enough?

WP Codex says crons should be scheduled on activation stuff, but this is tribe-common so that's not necessarily an option.